spring-cloud-azure-dependencies/pom.xml

```diff
@@ -26,16 +26,15 @@
         <commons.io.version>2.6</commons.io.version>
         <hibernate.validator.version>6.0.12.Final</hibernate.validator.version>
         <google.guava.version>20.0</google.guava.version>
-        <azure.sdk.version>1.14.0</azure.sdk.version>
+        <azure.sdk.version>1.23.0</azure.sdk.version>
```

Comment:
Is old version blocking this feature? If you update this, you need to do full integration tests of all modules.

Reply:
yes, old version doesn't support clientid when user-assigned identity, and according to their document:
>
clientid | Query | (Optional) The ID of the user-assigned identity to be used. If omitted, the system-assigned identity is used.
-- | -- | --

Reply:
sure, I will run full integration test